templatetags/chiffee_tags.py

```diff
@@ -0,0 +1,8 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter(name='has_group')
+def has_group(user, group_name):
+    return user.groups.filter(name=group_name).exists()
```

Comment:
What is this for?

Reply:
It's a custom tag for Django templates, used in making purchases when users are divided into groups.